src/Doctrine/ORM/Query/AST/Functions/Levenshtein.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Doctrine\ORM\Query\AST\Functions;
+
+use Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use Doctrine\ORM\Query\Lexer;
+use Doctrine\ORM\Query\Parser;
+use Doctrine\ORM\Query\SqlWalker;
+
+// Took from https://github.com/fza/mysql-doctrine-levenshtein-function#define-mysql-functions, thanks to him !
+class Levenshtein extends FunctionNode
+{
+    public ?object $firstStringExpression = null;
+    public ?object $secondStringExpression = null;
+
+    public function getSql(SqlWalker $sqlWalker)
+    {
+        return 'levenshtein('.
+            $this->firstStringExpression->dispatch($sqlWalker).', '.
+            $this->secondStringExpression->dispatch($sqlWalker).
+            ')';
+    }
```

Comment:
The CS is a bit ugly, could you format it nicely? I guess on one line with sprintf will be OK